tests/test_geom.py

```diff
@@ -13,3 +14,32 @@ def test_projection(tmp_path):
     ras_hdf = RasGeomHdf(tmp_path / "test.hdf")
     # Test the projection
     assert ras_hdf.projection() == CRS.from_wkt(wkt)
+
+def test_mesh_area_names(test_data_root = os.path.dirname(__file__).replace("tests", "data")):
```

Comment:
I think we should move `data` into the `tests` folder.

Reply:
Re: `test_data_root`, when `pytest` runs a unit test function it uses the first argument to provide a path to a temporary directory which is then removed. So the way that the tests are structured here, that feature isn't really getting used (and not really needed since the tests aren't writing any data to disk). I'd probably rewrite like this:

```python
from pathlib import Path

TEST_DATA = Path("./tests/data")  # assuming that 'data' is moved into 'tests`

def test_mesh_area_names():
    geom = TEST_DATA / "2d_geom/Richland_Lower.g01.hdf"
    with open RasGeomHdf(geom) as ghdf:
        assert ghdf.mesh_area_names() == ["NorthOverbank", "SouthOverbank"]
```

Reply:
Moved `data` to the `tests` folder and updated path handling as suggested.